src/rules/converters/codelyzer/component-class-suffix.ts

```diff
@@ -0,0 +1,15 @@
+import { RuleConverter } from "../../converter";
+
+export const convertComponentClassSuffix: RuleConverter = (tslintRule) => {
+    return {
+        rules: [
+            {
+                ...(tslintRule.ruleArguments.length !== 0 && {
+                    ruleArguments: tslintRule.ruleArguments,
```

Comment:
The [options](https://github.com/angular-eslint/angular-eslint/blob/e3a5796f28b20a6b6e64ba752f5a2d351b778109/packages/eslint-plugin/tests/rules/component-class-suffix.test.ts#L55) in angular-eslint look like they use an object with a `suffixes` member.

Reply:
Done! @JoshuaKGoldberg you're up to review again :)